site/public/templates/grading/TotalScoreBox.twig

```diff
@@ -5,34 +5,65 @@ Required Parameters:
 -auto_grading_total
 -ta_grading_earned
 -ta_grading_total
-
+-see_peer_grade
+-user_group
+l
 #}
-{% set show_total = auto_grading_total is defined and ta_grading_total is defined %}
-{% if auto_grading_total is defined %}
-    <div class="box-title badge-container">
-        <strong id="autograding_total" class="badge {{ functions.getBadgeStyle(auto_grading_earned, auto_grading_total, auto_grading_complete) }}">
-            {{ auto_grading_earned is defined ? auto_grading_earned|round(decimal_precision) : '&minus;' }} / {{ auto_grading_total|round(decimal_precision) }}
+{% if user_group < 4 %}
+    {% set show_total = auto_grading_total is defined and ta_grading_total is defined %}
```

Comment:
Total should be shown even if there is no autograding. I think it should always be shown for any non-peer grader.

Reply:
Updated this!